packages/editor/src/components/parts/common/eventcode/EventCodeSelect.tsx

```diff
@@ -29,7 +31,18 @@ const EventCodeSelect = ({ eventCode, onChange, eventCodes, eventIcon, comboboxI
     );
   };
 
-  return <Combobox value={eventCode} onChange={onChange} items={eventCodes} comboboxItem={comboboxItem} {...comboboxInputProps} />;
+  return withBrowser === false || withBrowser === undefined ? (
```

Comment:
withBrowser = undefined and withBrowser = false results in the same, so just turn it to make it simpler:

```ts
return withBrowser ? (
    <ComboboxInputWithBrowser
      browsers={['attr']}
      value={eventCode}
      onChange={onChange}
      items={eventCodes}
      comboboxItem={comboboxItem}
      {...comboboxInputProps}
    />
  ) : (
    <Combobox value={eventCode} onChange={onChange} items={eventCodes} comboboxItem={comboboxItem} {...comboboxInputProps} />
  );
```